certified-connectors/TPS & CTPS Checker/apiDefinition.swagger.json

```diff
@@ -0,0 +1,167 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "API for Querying the TPS/CTPS status of a UK telephone number",
+    "title": "TPS & CTPS Checker",
+    "version": "1.0",
+    "contact": {
```

Comment:
Kindly add URL into the contact section. Please refer this link. https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json#L9

Reply:
Completed as requested